### PR TITLE
Update the nit in the tutorial 

### PR DESCRIPTION
**Pull request by areshand**
## Motivation

update a nit 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Eyeball

### CODE REVIEW

**Review by JoshLind — APPROVED** [review]

File: developer-docs-site/docs/basics/basics-accounts.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ An account may contain an arbitrary number of Move resources and Move modules.
 
 ## Initial Account Setup
 
-An Aptos account is referenced by an account address, which is a 16-byte value. The account address is derived from the initial public verification key(s) for that account. Specifically, the account address is the last 16-bytes of the SHA-3 256 cryptographic hash of the initial public verification key(s) concatenated with a signature scheme identifier byte. The Aptos Blockchain supports two signature schemes: [Ed25519](/reference/glossary#ed25519) and MultiEd25519 (for multi-signature transactions). The account's private key is necessary for signing transactions.
+An Aptos account is referenced by an account address, which is a 32-byte value. The account address is derived from the initial public verification key(s) for that account. Specifically, the account address is the last 16-bytes of the SHA-3 256 cryptographic hash of the initial public verification key(s) concatenated with a signature scheme identifier byte. The Aptos Blockchain supports two signature schemes: [Ed25519](/reference/glossary#ed25519) and MultiEd25519 (for multi-signature transactions). The account's private key is necessary for signing transactions.
```
Comment (JoshLind) [inline developer-docs-site/docs/basics/basics-accounts.md:18]: Actually, there's another reference to 16 bytes still left in here. Can you update that?

**Review by JoshLind — DISMISSED** [review]

File: developer-docs-site/docs/basics/basics-accounts.md (hunk 1 above)
Comment (alinush) [reply developer-docs-site/docs/basics/basics-accounts.md:18]: Right, there are two contradictory sentences: "an account address, which is a 32-byte value" and "the account address is the last 16-bytes"

Comment (areshand) [reply developer-docs-site/docs/basics/basics-accounts.md:18]: just see the comments. will update soon

**Comment by areshand** [comment]
@JoshLind  @alinush  a second look?

**Review by alinush — APPROVED** [review]

**Comment by areshand** [comment]
@JoshLind  seems your approval is required to land this PR

**Comment by gregnazario** [comment]
> @JoshLind  seems your approval is required to land this PR

You might be able to reset his approval

**Pull request closed without merging**